lib/add-in-repo-tests-to-host.js

```diff
@@ -1,14 +1,13 @@
 const path = require('path');
 const { existsSync } = require('fs');
-const { WatchedDir } = require('broccoli-source');
 const mergeTrees = require('broccoli-merge-trees');
 const Funnel = require('broccoli-funnel');
 const flattenAddons = require('./flatten-addons');
 
-const generateTreeFromPath = path =>
-  existsSync(path) ? new WatchedDir(path) : null;
+const generateTreeFromPath = (path, processTests) =>
+  existsSync(path) ? new Funnel(processTests(path)) : null;
```

Comment:
Is this funnel necessary? It isn't doing anything, and looks like it could be replaced with just `existsSync(path) ? processTests(path) : null`. Also, why remove `WatchedDir`? Was what it was doing now unnecessary?